CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## 3.3.0
+#### New Functionality
+
+- Can now request MarkLogic to compress the response for better performance using enableGzippedResponses (MarkLogic Server version 11.0.0 or higher needed)
```

Comment:
Just toss periods at the end of each of these (one of the documentation folks scolded me recently for not doing that, so I'm sensitive to it now).